File: gherkinexecutor/Feature_Examples/ResultValueInternal.java
package gherkinexecutor.Feature_Examples;
import java.util.*;
class ResultValueInternal{
     Integer sum;
     
    public static String toDataTypeString() {
        return "ResultValueInternal {"
        +"Integer " 
            + "} "; }  
    ResultValue toResultValue() {
        return new ResultValue(
        String.valueOf(sum)
        ); }
    public ResultValueInternal(
        Integer sum
        )  {
        this.sum = sum;
        }
    @Override
    public boolean equals (Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;
        ResultValueInternal _ResultValueInternal = (ResultValueInternal) o;
         return 
                ( _ResultValueInternal.sum.equals(this.sum))
             ;  }
    @Override
    public String toString() {
        return "ResultValueInternal {"
        +"sum = " + sum + " "
            + "} " + "\n"; }  
    }

File: gherkinexecutor/Feature_Examples/ResultValueInternal.py
class ResultValueInternal:
    def __init__(self, sum):
        self.sum = sum

    @staticmethod
    def to_data_type_string():
        return "ResultValueInternal { Integer }"

    def to_result_value(self):
        return ResultValue(str(self.sum))

    def __eq__(self, other):
        if self is other:
            return True
        if other is None or not isinstance(other, ResultValueInternal):
            return False
        return self.sum == other.sum

    def __str__(self):
        return f"ResultValueInternal {{ sum = {self.sum} }}\n"

# Assuming ResultValue class is defined elsewhere
